vector/src/Data/Vector/Fusion/Stream/Monadic.hs

```diff
@@ -263,9 +265,9 @@ last (Stream step t) = last_loop0 SPEC t
 
 infixl 9 !!
 -- | Element at the given position
-(!!) :: Monad m => Stream m a -> Int -> m a
+(!!) :: (HasCallStack, Monad m) => Stream m a -> Int -> m a
 {-# INLINE (!!) #-}
-Stream step t !! j | j < 0     = ERROR "!!" "negative index"
+Stream step t !! j | j < 0     = error $ "negative index" Prelude.++ show j
```

Comment:
```suggestion
Stream step t !! j | j < 0     = error $ "negative index (" Prelude.++ show j Prelude.++ ")"
```
Alternatively `"negative index: " Prelude.++ show j`, but as is, this prints for example `negative index-1` if j = -1.